src/pages/Entity/WorkOrderInfo.tsx

```diff
@@ -55,8 +60,13 @@ const WorkOrderInfo = ({
             >
                 <TagInfoWrapper>
                     <CollapsibleCard cardTitle="Description">
-                        <p>{workOrder?.description}</p>
+                        <Description>
+                            {workOrder?.description
+                                ? removeHtmlFromText(workOrder?.description)
+                                : 'There is no description present, for more information see the short description.'}
```

Comment:
@VRiveland Hva tror du om denne?

Reply:
"for more information" er kanskje ikke helt perfekt. Føles nesten som om det er info om hvorfor det ikke er en lang description i den korte, hvis du skjønner?
Hva med noe som: "A long description is not present. See the short description at the top of the page for all information available" eller lignende? @Noggling

Reply:
Den hørtes mye bedre ut ja ;)